### PR TITLE
Improvement to userguide/dockerrepos/ documentation

### PR DESCRIPTION
**Pull request by mikebrow**
User requested clarification to documentation in the section titled "Searching for images" on https://docs.docker.com/userguide/dockerrepos/ 
Improves documentation to resolve issue #9235 

Signed-off-by: Mike Brown brownwm@us.ibm.com

### CODE REVIEW

File: docs/userguide/dockerrepos.md [hunk 1]
```diff
@@ -68,16 +68,28 @@ a user's repository from the image name.
...
 
 You now have an image from which you can run containers.
 
+### Specific Versions or Latest
+Using `docker pull centos` is equivalent to using `docker pull centos:latest`.
+To pull an image that is not the default latest image you can be more precise
+with the name of the image that you want.
+
+For example, to pull version 5 of `centos` use `docker pull centos:centos5`.
+In this example, `centos5` is a tag or name of an image in the `centos`
```
Comment (SvenDowideit) [inline docs/userguide/dockerrepos.md]:
```
In this example, `centos5` is the tag labelling the image in the `centos`
```

**Comment by SvenDowideit** [comment]
LGTM tho - its only a minor tweak - it'd be nice to avoid using `name` for `tags`

**Comment by mikebrow** [comment]
@SvenDowideit 
Thanks for the comments.  I've amended the edit per your suggestion.
Cheers, Mike

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Pull request merged**